licenses/management/commands/publish.py

```diff
@@ -0,0 +1,172 @@
+import os
+import subprocess
+from argparse import ArgumentParser
+from shutil import rmtree
+
+from django.conf import settings
+from django.core.management import BaseCommand, CommandError
+from django_distill.distill import urls_to_distill
+from django_distill.errors import DistillError
+from django_distill.renderer import render_to_dir
+
+from cc_licenses.settings.base import TRANSLATION_REPOSITORY_DIRECTORY
+from licenses.utils import cleanup_current_branch_output, strip_list_whitespace
+
+
+def run_django_distill(obj: object):
```

Comment:
All of these methods that are taking ```obj``` as an argument because they need access to the Command object can be moved to be methods on the Command class, then ```obj``` can just be changed to ```self```.